connector/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/model/user_registry/UserExtras.java

```diff
@@ -0,0 +1,13 @@
+package it.pagopa.selfcare.dashboard.connector.rest.model.user_registry;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
```

Comment:
@KevinSi96 use @Data instead of @Getter and @Setter so Lombok will create the toString() method too